app/javascript/leftSidebar/TagsFollowed.jsx

```diff
@@ -1,11 +1,10 @@
-import { h } from 'preact';
+import { h, Fragment } from 'preact';
 import PropTypes from 'prop-types';
 import { tagPropTypes } from '../common-prop-types';
 
 export const TagsFollowed = ({ tags = [] }) => {
-  // TODO: Once we're using Preact X >, we can replace the containing <div /> with a Fragment, <Fragment></Fragment>
   return (
```

Comment:
suggestion/non-blocking - you could just return `tags.map...` without the `Fragment` tag - since the `Fragment` effectively disappears when rendered, the result is the same 🙂

Reply:
Preact throws an error when I take out the Fragment, so maybe I'm missing a step? :shrug:

Reply:
I didn't explain myself well! I mean like this:
```
export const TagsFollowed = ({ tags = [] }) =>
  tags.map((tag) => (
    <div
      key={tag.id}
      className="sidebar-nav-element"
      id={`sidebar-element-${tag.name}`}
    >
      <a
        title={`${tag.name} tag`}
        className="crayons-link crayons-link--block"
        href={`/t/${tag.name}`}
      >
        {`#${tag.name}`}
      </a>
    </div>
  ));
```